fe/fe-core/src/main/java/org/apache/doris/planner/IcebergScanNode.java

```diff
@@ -99,9 +98,6 @@ public String getNodeExplainString(String prefix, TExplainLevel detailLevel) {
         StringBuilder output = new StringBuilder();
         if (!isLoad()) {
             output.append(prefix).append("TABLE: ").append(icebergTable.getName()).append("\n");
-            output.append(prefix).append("PATH: ")
```

Comment:
Uri or warehouse information should be added here.

Reply:
Added